packages/component/src/SendBox/AttachmentBar/AttachmentBarStyle.ts

```diff
@@ -0,0 +1,40 @@
+import { StrictStyleOptions } from 'botframework-webchat-api';
+import { type StyleSet } from '../../Styles/StyleSet/types/StyleSet';
+
+export default function createSendBoxAttachmentBarStyle({ sendBoxAttachmentBarMaxHeight }: StrictStyleOptions) {
+  return {
+    '&.webchat__send-box-attachment-bar': {
+      display: 'content',
+      gridArea: 'attachment-bar',
```

Comment:
I don't think it'll follow the `grid-area` for `display: content`.

Reply:
Right... and it's `contents` but not `content`, so it's being ignored.